manim/stream_starter.py

```diff
@@ -0,0 +1,157 @@
+import code
+import functools
+import os
+import readline
+import rlcompleter
+import subprocess
+
+from colorama import Fore, Style
```

Comment:
why this is needed?
if for coloured output use rich.

Reply:
Fair enough. Except I only needed the syntax to print it as a nice green, so that didn't need that much effort

I'm open to suggestions though.

Reply:
Use `console.print` statements from rich. It can be coloured easily and is cross-platform. See how it is done in other places.